### PR TITLE
Add User club and UserVoice on BO link footer

### PR DESCRIPTION
**Pull request by sLorenzini**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.3.x
| Description?  | Add User club and UserVoice on BO link footer.
| Type?         | new feature
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | http://forge.prestashop.com/browse/FF-166
| How to test?  | Scroll to bottom page on Dashboard BO and see new links

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/8849)

### CODE REVIEW

**Comment by LittleBigDev** [comment]
:lgtm:
Straightforward.

---

Reviewed 1 of 1 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/prestashop/prestashop/8849#-:-L7Yz0XY7I5QUqR1_R1X:bl4tnjw)*

**Review by LittleBigDev — APPROVED** [review]
Thank you @sLorenzini

File: admin-dev/themes/default/template/footer.tpl [hunk 1]
```diff
@@ -62,6 +62,16 @@
...
 			</a>
 			/&nbsp;
+			<a href="https://www.prestashop.com/club/?utm_source=back-office&amp;utm_medium=footer&amp;utm_campaign=back-office-{$lang_iso|upper}&amp;utm_content={if $host_mode}cloud{else}download{/if}" class="footer_link _blank">
+				<i class="icon-users"></i>
+				{l s='User Club' d='Admin.Navigation.Footer'}
+			</a>
+			/&nbsp;
+			<a href="http://feedback.prestashop.com/forums/387864-prestashop-1-7-x?utm_source=back-office&amp;utm_medium=footer&amp;utm_campaign=back-office-{$lang_iso|upper}&amp;utm_content={if $host_mode}cloud{else}download{/if}" class="footer_link _blank">
+				<i class="icon-lightbulb"></i>
+				{l s='Feature Requests' d='Admin.Navigation.Footer'}
```
Comment (eternoendless) [inline admin-dev/themes/default/template/footer.tpl:72]: @sLorenzini @colinegin @Saycile I don't think these wordings exist in the 1.7.3 catalog.

We cannot add new ones until the next minor, so we have to either work around that (as in, hardcode it in English) or wait until 1.7.4.

**Comment by LittleBigDev** [comment]
ping @sLorenzini

**Comment by LittleBigDev** [comment]
ping @sLorenzini @colinegin @Saycile

**Comment by emmanuelf75009** [comment]
Hi, as said with @eternoendless, this field will be named "Feature Requests" as it is an international standard name in 1.7.3.x and will be translatable from 1.7.4

**Review by eternoendless — APPROVED** [review]

**Comment by eternoendless** [comment]
Thank you @sLorenzini

**Pull request merged**